### PR TITLE
Changes for upcoming IBM Z P pipelines support

### PR DESCRIPTION
**Pull request by SNiemann15**

### CODE REVIEW

File: modules/op-installing-tkn-on-linux-on-ibm-power.adoc
Comment (vdemeester) [inline modules/op-installing-tkn-on-linux-on-ibm-power.adoc:13]: This should be `0.13.1` instead.

File: modules/op-installing-tkn-on-linux-on-ibm-z.adoc
Comment (vdemeester) [inline modules/op-installing-tkn-on-linux-on-ibm-z.adoc:13]: This should be `0.13.1` instead.

**Review by vdemeester — COMMENTED** [review]
SGTM except the version
/cc @Preeticp

**Comment by vdemeester** [comment]
One thing to note here, 1.2 is currently only available in the `preview` channel, which is not the default on OCP 4.6. The default channel is `ocp-4.6` is 1.1.x which doesn't have P & Z builds. We may want to warn user that for now (until we get devconsole ack to move) that they will need to use `preview` to be able to install OpenShift Pipelines on P or Z.
/cc @Preeticp

File: modules/op-installing-tkn-on-linux-on-ibm-power.adoc
Comment (ppitonak) [inline modules/op-installing-tkn-on-linux-on-ibm-power.adoc:13]: correct version is 0.13.1

File: modules/op-installing-tkn-on-linux-on-ibm-z.adoc
Comment (ppitonak) [inline modules/op-installing-tkn-on-linux-on-ibm-z.adoc:13]: correct version is 0.13.1

File: modules/op-release-notes-1-2.adoc
Comment (ppitonak) [inline modules/op-release-notes-1-2.adoc:17]: @Preeticp We should probably mention that user need to install 1.2 from `preview` channel (and remove that note later when https://issues.redhat.com/browse/SRVKP-988 is resolved).

While it's confusing for x86 architecture (but installation from `ocp-4.6`  works, you only get older release), the installation from `ocp-4.6` channel on P&Z will fail because 1.1 doesn't exist on those architectures.

File: cli_reference/tkn_cli/installing-tkn.adoc
Comment (ppitonak) [inline cli_reference/tkn_cli/installing-tkn.adoc:16]: With this PR merged, we would have 3 sections that are basically same, the download link is the only difference. @Preeticp would it make sense to have just one section and listing all three tarballs there?

Comment (SNiemann15) [reply cli_reference/tkn_cli/installing-tkn.adoc:16]: @ppitonak We took the same approach as we did for odo:  https://docs.openshift.com/container-platform/4.6/cli_reference/developer_cli_odo/installing-odo.html
Not sure whats better.

File: modules/op-release-notes-1-2.adoc
Comment (Preeticp) [reply modules/op-release-notes-1-2.adoc:17]: @SNiemann15,  Pavol and Vincent are right, you will need to add a note for both power and Z. Also, please start watching the Jira so that you know when the Jira is resolved and you can update the note.

File: cli_reference/tkn_cli/installing-tkn.adoc
Comment (Preeticp) [reply cli_reference/tkn_cli/installing-tkn.adoc:16]: @ppitonak totally makes sense. 
@SNiemann15 if you see the preview generated here: https://z_p_pipelines--ocpdocs.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html
you will see he is right. There is no difference except the download link.
I suggest we simply retain the module: Installing Red Hat OpenShift Pipelines CLI (tkn) on Linux.
the first step could then be: 
1. Download the  relevant CLI.
    - Linux <download link>
    - Linux on IBM Z and LinuxONE <download link>
    - Linux on IBM Power Systems <download link>

I think we can do the same in odo too, but @boczkowska can take a call on that. Yana, can you please see if we can do something similar for odo?

File: modules/op-installing-tkn-on-linux-on-ibm-power.adoc
Comment (SNiemann15) [reply modules/op-installing-tkn-on-linux-on-ibm-power.adoc:13]: Is it 0.13.1 for both the folder and the tar file like this? 

https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.13.1/tkn-linux-ppc64le-0.13.1.tar.gz

Or only the tar?

File: modules/op-release-notes-1-2.adoc
Comment (Preeticp) [inline modules/op-release-notes-1-2.adoc:31]: @SNiemann15 I would put this note after line 23. The commented out part on L24 and 25 needs to be deleted.

File: modules/op-release-notes-1-2.adoc
Comment (Preeticp) [inline modules/op-release-notes-1-2.adoc:105]: - Suggest: When you run...
- As per the [latest guidelines on OCP](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references): s/ Task/`Task` resource
- ...without a command ...
- IBM Power Systems/IBM Power System ?
- ...cluster, the `TaskRun` resource fails...

File: modules/op-release-notes-1-2.adoc
Comment (Preeticp) [inline modules/op-release-notes-1-2.adoc:112]: ...specify the sha256 digest...

**Review by Preeticp — CHANGES_REQUESTED** [review]
Hi Silke, apart from Pavel's comment on unifying the modules I have left some other minor comments, please fix them.

File: cli_reference/tkn_cli/installing-tkn.adoc
Comment (SNiemann15) [reply cli_reference/tkn_cli/installing-tkn.adoc:16]: @ppitonak @Preeticp Seems like an additional step is needed for P and Z. Should we still try to keep it in one procedure? 

after testing both @mohit-sheth @xzheng@redhat.com said today that as step  2 `$ chmod +x tkn-linux-s390x`
is needed.

Comment (ppitonak) [reply cli_reference/tkn_cli/installing-tkn.adoc:16]: @SNiemann15 thank you for letting us know... this step is now necessary also for x86 binaries. I'll try to find out if it would be possible to revert to old behavior

**Comment by zhengxiaomei123** [comment]
@zhengxiaomei123

File: modules/op-installing-tkn-on-linux-on-ibm-power.adoc
Comment (zhengxiaomei123) [reply modules/op-installing-tkn-on-linux-on-ibm-power.adoc:13]: > Is it 0.13.1 for both the folder and the tar file like this?
> 
> https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.13.1/tkn-linux-ppc64le-0.13.1.tar.gz
> 
> Or only the tar?

both the folder

**Review by tgargya — COMMENTED** [review]
For Z/P /lgtm

**Review by tgargya — COMMENTED** [review]
/lgtm - for Z and P

**Comment by SNiemann15** [comment]
@ppitonak @Preeticp Please have a look. I'll remove "WIP"

File: modules/op-release-notes-1-2.adoc
Comment (ppitonak) [reply modules/op-release-notes-1-2.adoc:16]: Today we talked about promoting 1.2.1 into `ocp-4.6` channel so it maybe this note will not be necessary. Still waiting for approval

File: modules/op-installing-tkn-on-linux.adoc
Comment (ppitonak) [inline modules/op-installing-tkn-on-linux.adoc:15]: should we specify the architecture? (something like x86_64 or amd64)

Comment (SNiemann15) [reply modules/op-installing-tkn-on-linux.adoc:15]: @ppitonak @Preeticp How about: 

Download the relevant CLI.

-     Linux (x86, amd64) 
-     Linux on IBM Z and LinuxONE (s390x)
-     Linux on IBM Power Systems (ppc64le)

Comment (ppitonak) [reply modules/op-installing-tkn-on-linux.adoc:15]: x86 usually refers to 32-bit architecture which is incompatible, it should be `x86_64`

Comment (Preeticp) [reply modules/op-installing-tkn-on-linux.adoc:15]: +1 this makes sense

Comment (SNiemann15) [reply modules/op-installing-tkn-on-linux.adoc]: @Preeticp Added the architectures

File: modules/op-release-notes-1-2.adoc
Comment (Preeticp) [inline modules/op-release-notes-1-2.adoc:109]: multi-arch container image `Task`/multi-arch container image task

**Comment by Preeticp** [comment]
@SNiemann15 apart from the one comment about changing task to small case and removing backticks this looks good to me.
Please ask the OCP dosc team's  peer review squad in the forum channel to review and merge it.

File: modules/op-release-notes-1-2.adoc
Comment (ahardin-rh) [reply modules/op-release-notes-1-2.adoc:16]: Instead of RHOCP I would use {product-title}

Right now, we don't commonly use RHOCP in our doc set, though that might change. There were some recent discussions about that. For now, I would avoid its usage.

**Comment by ahardin-rh** [comment]
@SNiemann15 Just one minor comment from me. Otherwise, looks great!

**Pull request merged**
